feat : live pairs

MikeMurpher opened the pull request.

cryptod00c commented:
@MikeMurpher - pushed some updates to this PR - nice work!

Is all we can get the contract address?

ideally, it'd be great to get the `Ticker` and what chain it's on...

additionally, we'll probably have to host an external API endpoint - only because we'll have to make the call serverside to fetch all the live chains - searching for available pairs...and return it to the frontend...

I don't see any way around fetching for 5 chains with a local one chain RPC.

This should only have one chart Icon - (once we can determine the chain) and the ticker and we should be good to go.

![Screenshot 2023-02-21 at 22 39 38](https://user-images.githubusercontent.com/104535928/220515573-22b14801-fd61-47a0-b194-978116659e7f.png)

MikeMurpher commented:
Definitely can but thought it was best to see the structure of the table first. I've now added these details.

I've updated the icons and Xchange links for user ability so we can have an up-and-running live listing where users only click the valid links. ETH is the default and table-loading factory pair based on the wagmi chain connection.
